app/usedgoodreads/__main__.py

```diff
@@ -27,17 +29,19 @@ def index(key):
         books = UsedBook.get(isbn=cache.isbn)
         isbn = cache.isbn
         title = cache.title
+
         if books:
             # Used books are in database
             results = [{"title": v.title, "description": v.description,
-                        "price": v.price, "link": v.link} for v in books]
-            results.sort(key=lambda each: each["price"])
+                        "price": v.price if not math.isnan(v.price) else None, "link": v.link} for v in books]
+            results.sort(key=lambda each: each["price"] if each["price"] != None else float("inf"))
```

Comment:
- Why don't we use `None` already at parsing the file instead of `float("NaN")`? Do we have to change the schema to allow this?

Reply:
https://github.com/usedgoodreads/usedgoodreads/blob/665b1a70f09eeb9c8084d602cb9c18cc6bbf68f9/app/usedgoodreads/models/usedbook.py#L11 it's a non-nullable float right now; we can make it nullable and then do what you said